replica/rest/model.go

```diff
@@ -180,9 +181,6 @@ func NewReplica(context *api.ApiContext, state replica.State, info replica.Info,
 		actions["setreplicacounter"] = true
 		actions["updatecloneinfo"] = true
 	case replica.Rebuilding:
-		actions["start"] = true
```

Comment:
can we remove many other actions from Rebuilding state? (or) do we need to allow them so that if replica restart happens after the state is set?

Reply:
these actions are required during rebuild